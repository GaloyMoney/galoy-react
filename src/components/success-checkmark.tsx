export const SuccessCheckmark = () => {
  return (
    <div className="success-checkmark">
      <div className="check-icon">
        <span className="icon-line line-tip" />
        <span className="icon-line line-long" />
        <div className="icon-circle" />
        <div className="icon-fix" />
      </div>
    </div>
  )
}
