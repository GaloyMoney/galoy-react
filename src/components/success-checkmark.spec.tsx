import { render } from "@testing-library/react"

import { SuccessCheckmark } from "./success-checkmark"

describe("SuccessCheckmark", () => {
  it("renders and matches snapshot", () => {
    const { asFragment } = render(<SuccessCheckmark />)
    expect(asFragment()).toMatchSnapshot()
  })
})
