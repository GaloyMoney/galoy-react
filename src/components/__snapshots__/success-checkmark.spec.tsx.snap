// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuccessCheckmark renders and matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="success-checkmark"
  >
    <div
      class="check-icon"
    >
      <span
        class="icon-line line-tip"
      />
      <span
        class="icon-line line-long"
      />
      <div
        class="icon-circle"
      />
      <div
        class="icon-fix"
      />
    </div>
  </div>
</DocumentFragment>
`;
